service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/validator/EPORemovalAddressValidator.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.service.orders.validator;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.event.ManageOrdersEventData;
+import uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.logging.log4j.util.Strings.isEmpty;
+import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.EPO_PREVENT_REMOVAL;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class EPORemovalAddressValidator implements QuestionBlockOrderValidator {
+
+    private static final String INVALID_ADDRESS_LINE_MESSAGE = "Enter a valid address for the contact";
+    private static final String INVALID_POST_CODE_MESSAGE = "Enter a postcode for the contact";
+
+    @Override
+    public OrderQuestionBlock accept() {
+        return EPO_PREVENT_REMOVAL;
+    }
+
+    @Override
+    public List<String> validate(CaseData caseData) {
+        final ManageOrdersEventData eventData = caseData.getManageOrdersEventData();
+        final Address removalAddress = eventData.getManageOrdersEpoRemovalAddress();
```

Comment:
The address is a mandatory field, why are we validating that it is empty or are we not trusting xui?

Reply:
I added it assuming we need postcode validation. UX confirmed that we don't need postcode validation, so I'll remove this class and tests